iotedgedev/connectionstring.py

```diff
@@ -20,6 +24,22 @@ def __init__(self, value):
     def __getitem__(self, key):
         return self.data[key]
 
+    @suppress_all_exceptions()
+    def hash_hostname(self):
```

Comment:
I was thinking this could be calculated when the connection string was initially parsed.

so you can just access it with `connection_string.HostNameHashed`

Reply:
Updated. Should look better now.